### PR TITLE
Update plugins.json

### PR DESCRIPTION
**Pull request by DiekoMA**
Added Godot plugin

### CODE REVIEW

**Review by VictoriousRaptor — DISMISSED** [review]

**Review by jjw24 — CHANGES_REQUESTED** [review]
Minor changes

File: plugins.json [hunk 1]
```diff
@@ -1908,5 +1908,19 @@
...
+        "Description": "Quickly launch your Godot projects from flow launcher",
+        "Author": "DiekoMA",
+        "Version": "1.0.0",
+        "Language": "csharp",
+        "Website": "https://github.com/DiekoMA/Flow.Launcher.Plugin.Godot",
+        "IcoPath": "https://cdn.jsdelivr.net/gh/DiekoMA/Flow.Launcher.Plugin.Godot@master/Assets/Godot.png",
+        "UrlSourceCode": "https://github.com/DiekoMA/Flow.Launcher.Plugin.Godot/tree/master",
+        "UrlDownload": "https://github.com/DiekoMA/Flow.Launcher.Plugin.Godot/releases/tag/v1.0.0/Godot.zip",
+        "DateAdded": "2023-09-20T14:33:01Z",
+        "LatestReleaseDate": "2023-09-20T14:33:01Z"
```
Comment (jjw24) [inline plugins.json:1924]:
```suggestion
        "UrlDownload": "https://github.com/DiekoMA/Flow.Launcher.Plugin.Godot/releases/tag/v1.0.0/Godot.zip"
```

Comment (jjw24) [reply plugins.json:1924]: DateAdded and LatestReleaseDate fields are auto-populated.

File: plugins.json [hunk 2]
```diff
@@ -1908,5 +1908,19 @@
...
+        "ID": "D8AF059F555C48A380AAD5D57B4A5D61",
+        "Name": "Godot",
+        "Description": "Quickly launch your Godot projects from flow launcher",
+        "Author": "DiekoMA",
+        "Version": "1.0.0",
+        "Language": "csharp",
+        "Website": "https://github.com/DiekoMA/Flow.Launcher.Plugin.Godot",
+        "IcoPath": "https://cdn.jsdelivr.net/gh/DiekoMA/Flow.Launcher.Plugin.Godot@master/Assets/Godot.png",
+        "UrlSourceCode": "https://github.com/DiekoMA/Flow.Launcher.Plugin.Godot/tree/master",
+        "UrlDownload": "https://github.com/DiekoMA/Flow.Launcher.Plugin.Godot/releases/tag/v1.0.0/Godot.zip",
```
Comment (jjw24) [inline plugins.json:1922]: I don't see a GitHub workflow file in your repo, are you uploading this zip by hand?

Comment (DiekoMA) [reply plugins.json:1922]: yes

Comment (jjw24) [reply plugins.json:1922]: Could we ask you to use GitHub Actions to publish your plugin please. The reason is packaging and uploading the zip manually is hard for us to make reasonable judgement that what's contained in the zip is the same from the repository.

You can follow this as the example, most likely just need to replace the dotnet version and the name of the plugin: https://github.com/taooceros/Flow.Plugin.WindowWalker/blob/master/.github/workflows/publish-action.yml

Comment (DiekoMA) [reply plugins.json:1922]: sure thing

Comment (DiekoMA) [reply plugins.json:1922]: i've set up the workflow

Comment (jjw24) [reply plugins.json:1922]: Hmm for some reason you are getting 403 on publish, something to do with the token you defined in the workflow maybe - https://github.com/DiekoMA/Flow.Launcher.Plugin.Godot/blob/5f7f7dcbc6fe9fbf9e768351cc18f69385ea379b/.github/workflows/main.yml#L37

Maybe try setting your own token perhaps.

Comment (DiekoMA) [reply plugins.json:1922]: I can use a personal access token with permissions for actions right ?

Comment (jjw24) [reply plugins.json:1922]: Yep

Comment (DiekoMA) [reply plugins.json:1922]: still didn't run, it also said i can't name a secret GITHUB_TOKEN

Comment (jjw24) [reply plugins.json:1922]: Change the secret name from GITHUB_TOKEN to a different one, must be the same as the token name you added in the GitHub settings page

Comment (DiekoMA) [reply plugins.json:1922]: It's working properly now, thank you

Comment (jjw24) [reply plugins.json]: No worries, thanks for fixing it up. Will take a look later when in front of computer.

File: plugins.json [hunk 3]
```diff
@@ -1919,5 +1919,17 @@
...
+        "ID": "D8AF059F555C48A380AAD5D57B4A5D61",
+        "Name": "Godot",
+        "Description": "Quickly launch your Godot projects from flow launcher",
+        "Author": "DiekoMA",
+        "Version": "1.0.0",
+        "Language": "csharp",
+        "Website": "https://github.com/DiekoMA/Flow.Launcher.Plugin.Godot",
+        "IcoPath": "https://cdn.jsdelivr.net/gh/DiekoMA/Flow.Launcher.Plugin.Godot@master/Assets/Godot.png",
+        "UrlSourceCode": "https://github.com/DiekoMA/Flow.Launcher.Plugin.Godot/tree/master",
+        "UrlDownload": "https://github.com/DiekoMA/Flow.Launcher.Plugin.Godot/releases/tag/v1.0.0/Godot.zip"
```
Comment (jjw24) [inline plugins.json:1933]:
```suggestion
        "Author": "DiekoMA",
        "Version": "1.0.1",
        "Language": "csharp",
        "Website": "https://github.com/DiekoMA/Flow.Launcher.Plugin.Godot",
        "IcoPath": "https://cdn.jsdelivr.net/gh/DiekoMA/Flow.Launcher.Plugin.Godot@master/Assets/Godot.png",
        "UrlSourceCode": "https://github.com/DiekoMA/Flow.Launcher.Plugin.Godot/tree/master",
        "UrlDownload": "https://github.com/DiekoMA/Flow.Launcher.Plugin.Godot/releases/download/v1.0.1/Godot-1.0.1.zip"
```

**Review by jjw24 — APPROVED** [review]

**Pull request merged**